app/routes/__frontend/resources/developer-resources/developer-health/burnout.mdx

```diff
@@ -0,0 +1,101 @@
+
+Sometimes you must step away from the things you can no longer sustain. You may have taken on too much, or you changed your priorities. That’s ok. Learning to pause and step away from things is important. Trying to do too much will accelerate burnout.
+
+</TextContainer>
+
+<TextContainer>
+
+## Increasing Positive Influences
+
+Counteracting the negative influences in your life by investing in the people and things that bring you energy or happiness can help to navigate burnout. Find the activities that bring you joy and comfort, and incorporate those into your week. Put them on your schedule as you would work meetings or interviews to signify their importance and necessity.
+
+Sometimes, physically staying in the same place increases feelings of burnout. Try moving your workspace. As part of this, incorporating regular body movement can decrease feelings of being trapped. It might help to set some goals to walk X every week, start a 100-day push-up challenge, or set up a physical goal that holds you accountable but also accomplished.
+
+Create adequate time to wind down every day. This will also contribute to much better sleep, which is essential for burnout recovery.
+
+Consider making space in the week to focus on activities that bring you joy. This might be playing games, watching movies, hanging out with friends and family, heading to the gym, or doing any activity that gives you energy. Along with that, it’s ok and good to rest. Finding space not to do things is equally essential in replenishing your energy.
+
+Let yourself be bored! Daydream, dream about new projects, think abstractly about concepts you’ve learned, identify new connections between parts of yourself, reflect on what learning approaches worked well, and even pursue some opportunities you hadn’t considered.
+
+</TextContainer>
+
+<TextContainer>
+
+## Create Boundaries
+
+Boundaries help to keep us healthy, but often we find that we’ve let things cross our boundaries or the boundaries have blurred. You may have worked late one week to hit a big goal for your job. Suddenly, you find that you’re being asked to do that—or worse, you’re being expected to do it without your consent—for everyday tasks. It’s not in your contract, and you value work-life harmony.
+
+Regularly assessing your upcoming week and retrospect to identify the biggest stressors can help prevent this from continuing. You may identify ways to improve your stressors by reducing hours, taking more breaks, setting a day without meetings, or finding space to work in an environment conducive to doing your best work. Consider doing a self-reflection as well.
+
+To assess your week, set a stop time for every day. Although it may be necessary to push past that in some instances, it’s important to make that the exception and not the rule.
+
+</TextContainer>
+
+<TextContainer>
+
+## Affirmations
+
```

Comment:
Identify affirmations that feel authentic to you such as:

- I matter.
- My worth isn't tied to my productivity, side projects, or job title.
- I am more than my rejections.
- I am important.

Then, create a daily self-affirmation habit like repeating these affirmations for 3 - 5 minutes once or twice a day. If affirmations feel inauthentic to you, try writing down one thing you're grateful for every day. These habits are to help you remember that you have things you cherish and your life is multi-dimensional.